autonomy/cli/scaffold_fsm.py

```diff
@@ -371,7 +372,7 @@ def _update_behaviours(self, config: SkillConfig) -> None:
         """Update the behaviours section of the skill configuration."""
         config.behaviours = CRUDCollection[SkillComponentConfiguration]()
         abci_app_cls_name = self.dfa.label.split(".")[-1]
-        round_behaviour_cls_name = abci_app_cls_name.replace(ROUND, BEHAVIOUR)
+        round_behaviour_cls_name = abci_app_cls_name.replace(ABCI_APP, ROUND_BEHAVIOUR)
```

Comment:
The `RoundBehaviour{}` is the AEA-behaviour, not the `AbciApp{}`.

Reply:
Aah I missed this. Only correct in template kwargs last PR. This has been incorrect from the get-go
https://github.com/valory-xyz/open-autonomy/blob/ebf69d0e2c08c39b1e54485196909860034d3dce/autonomy/cli/scaffold_fsm.py#L196